### PR TITLE
fix: import theme file breaks custom theme dialog

### LINKED ISSUE #2923: Importing theme file breaks customTheme dialog
**Describe the bug**
When we import a custom Theme file, we cannot change the color Input values

**To Reproduce**
1. Go to circuitverse simulator
2. Click on tools > themes > customThemes
3. Import a theme file
4. try to change color
5. see error

**Expected behavior**
After importing the theme file, the user is able to change input color values

### PR DESCRIPTION
**Pull request by Jaagrav**
Fixes #2923 

#### Changes I made in this PR -
The issue was caused because the event listener responsible to update the colors/theme on the simulator page was stopping to work after importing a json file for setting a custom theme. Therefore I added the event listener inside another function which is called automatically when the exported function `CustomColorThemes` is called in order to set the event listener. Now I am using this same function signature to call and reset the event listener every time a new custom theme file is imported, inside the function `receivedText`

### Screencast -
Basically proof that it works...

https://user-images.githubusercontent.com/52719271/170951656-2c6edba4-5d4d-416f-b6a2-4990f2db715f.mp4

### CODE REVIEW

**Review by pulkit-30 — APPROVED** [review]
Hey @Jaagrav, Welcome to CircuitVerse 🎉 
Thank you so much. This looks great!

**Review by gr455 — APPROVED** [review]
